__tests__/app.test.js

```diff
@@ -33,3 +33,25 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe.only("GET /api/reviews", () => {
+  test("200: returns a reviews array of review objects, sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeSortedBy("created_at", { descending: true });
+        expect(reviews).toHaveLength(13);
```

Comment:
add a couple of assertions here just to check that the objects in the array are how we expect them to look, i.e. they have all the properties we want such as category, designer etc.